File: benchmark/array4b.pl
#!/usr/local/bin/perl

$base = 8000;

for ($i=0; $i<$base; $i++) {
	$si = "".$i;
	$arr1{$si} = $base - $i;
}
for ($i=5; $i<$base; $i++) {
	$ind = "".($i-1);
	$si = "".$i;
	$arr1{$ind} = $arr1{$si};
}

File: benchmark/array4b.py
#!/usr/local/bin/python3

base = 8000

arr1 = {} 
for i in range(base):
    si = str(i)
    arr1[si] = base - i

for i in range(5,base):
    sind = str(i-1)
    si = str(i)
    arr1[sind] = arr1[si];

